study-builder/src/main/java/org/broadinstitute/ddp/studybuilder/task/AddStateOption.java

```diff
@@ -0,0 +1,85 @@
+package org.broadinstitute.ddp.studybuilder.task;
```

Comment:
One time run in each env

Reply:
Is it possible to prevent occasional run more than 1 time?

Reply:
Running more than once will fail because we will run into UK violation (same stableID)